content/accessibility/designer-toolkit.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Designer toolkit
+---
+
+A collection of tools to help product designers design accessibly. 
+
+Whether you’re designing a new user experience or just one component, make sure you’re addressing the considerations from the respective list.
+
+## Accessibility checklists
+
+### Interactive control creation
+If you're designing a new component or control for GitHub that will not be made up of one single piece of [standard semantic markup](https://developer.mozilla.org/en-US/docs/Web/HTML/Element), check out the Accessibility Portal's [custom control design checklist](https://www.a11yportal.com/guidelines/fundamentals/checklists.html#custom-control-checklist-ux).
+
+### UX flow
```

Comment:
I still don't understand what this title means, can we make it more descriptive, and add a sentence below it that introduces the list?

Reply:
### New or updated feature
If you're designing a feature or experience in GitHub, use this checklist below to assess your design for accessibility before developer hand-off.